### PR TITLE
Add instructions for widescreen support on Storyboarding

### PR DESCRIPTION
**Pull request by Awears**

### CODE REVIEW

File: wiki/Storyboarding/en.md [hunk 1]
```diff
@@ -6,6 +6,6 @@
 
 ## Creating a storyboard
 
-The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of simple sprites, but is often regarded as clunky and unusable for anything beyond the most basic of storyboards. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog.
+The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of sprites, but is often regarded as clunky and unusable for anything beyond the basics of storyboarding. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog, then go to the last page, then check the box that says 'Widescreen Support'.
```
Comment (bdach) [inline wiki/Storyboarding/en.md]: Tense mismatch at the end there:

> (...) by *going* to the song setup dialog, then *go* to the last page, (...)

File: wiki/Storyboarding/en.md [hunk 2]
```diff
@@ -6,6 +6,6 @@
...
 
-Users skilled in storyboard creation often use the backend code created by the SBE, entitled [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) instead, as it affords them more control. See that article for details.
+Users skilled in storyboard creation often use the backend code created by the SBE, entitled [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) instead, as it offers them more control. See the article for details.
```
Comment (bdach) [inline wiki/Storyboarding/en.md]: The article change feels more confusing:

> See the article for details.

*Which* article? The previous version avoided that by explicitly referring to the previous sentence.

Comment (Awears) [reply wiki/Storyboarding/en.md]: I reverted the change

File: wiki/Storyboarding/en.md (hunk 1 above)
Comment (Awears) [reply wiki/Storyboarding/en.md]: Oops. I made it say 'go to the design tab' now.

File: wiki/Storyboarding/en.md [hunk 3]
```diff
@@ -6,6 +6,6 @@
 
 ## Creating a storyboard
 
-The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of simple sprites, but is often regarded as clunky and unusable for anything beyond the most basic of storyboards. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog.
+The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of sprites, but is often regarded as clunky and unusable for anything beyond the basics of storyboarding. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog, go to the Design tab, then check the box that says 'Widescreen Support'.
```
Comment (bdach) [inline wiki/Storyboarding/en.md]: This sentence still doesn't read correctly. It starts with a continuous infinitive (*going*) and then has simple infinitives after (*go*, *check*).

Comment (Awears) [reply wiki/Storyboarding/en.md]: i applied your suggestion

**Pull request merged**